### PR TITLE
Allow skipping incineration of processed emails

### PR DESCRIPTION
**Pull request by lifo**
### Summary

Allow skipping incineration of processed emails. This would be useful when applications wants to keep the original email around and manage its incineration.

### CODE REVIEW

File: actionmailbox/lib/action_mailbox.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ module ActionMailbox
 
   mattr_accessor :ingress
   mattr_accessor :logger
+  mattr_accessor :skip_incineration, default: false
```
Comment (georgeclaghorn) [inline actionmailbox/lib/action_mailbox.rb]: Can we name this option `incinerate` and make the default value `true`? Then you’d disable incineration by setting it to false.

**Review by georgeclaghorn — CHANGES_REQUESTED** [review]
I knew this PR would come eventually. :smile:

`ActionMailbox::Engine` needs to be updated to set the new option from application config.

**Comment by lifo** [comment]
@georgeclaghorn Added it to `ActionMailbox::Engine`

File: actionmailbox/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 ## Rails 6.0.0.beta1 (January 18, 2019) ##
 
+*   Allow skipping incineration of processed emails.
+
+    This can be done by setting `config.action_mailbox.incinerate` to `false`.
+
+    *Pratik Naik*
+
```
Comment (georgeclaghorn) [inline actionmailbox/CHANGELOG.md]: Last thing: this should go above the beta1 heading. beta1 already shipped.

Comment (lifo) [reply actionmailbox/CHANGELOG.md]: Ah, missed that. Pushed a fix.

**Pull request merged**